### PR TITLE
Pod fix

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: utility/podfile_util_test.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	}
+
+	t.Log("workspace defined, two project in the pod folder -- ERROR")
+	{
+		tmpDir, err := pathutil.NormalizedOSTempDirPath("__utility_test_3__")
+		require.NoError(t, err)
+
+		podfile := `platform :ios, '9.0'
+pod 'Alamofire', '~> 3.4'
+workspace 'MyWorkspace'
```
Comment (viktorbenei) [inline utility/podfile_util_test.go]: add test for `workspace "MyWorkspace"` - maybe even with "smart quotes"

File: utility/podfile_util.go [hunk 2]
```diff
@@ -182,9 +190,58 @@ workspaces = podfile.get_workspaces(podfile.list_targets.first)
 puts workspaces.to_json
 `
 
-// GetWorkspaces ...
-func GetWorkspaces(searchDir string) (map[string]string, error) {
-	tmpDir, err := pathutil.NormalizedOSTempDirPath("bitrise-plugin-init")
+func isWorkspaceSpecified(podfileContent string) bool {
+	re := regexp.MustCompile("^workspace (.+)")
```
Comment (godrei) [inline utility/podfile_util.go]: prepare for:
`workspace MyWorkspace`
`  workspace MyWorkspace`

File: utility/podfile_util_test.go [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+	"strings"
+	"testing"
+
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsWorkspaceSpecified(t *testing.T) {
+	t.Log("podfile - no workspace defined")
```
Comment (godrei) [inline utility/podfile_util_test.go]: add test for there is a space before workspace tag

File: utility/podfile_util_test.go [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+		require.Equal(t, true, isWorkspaceSpecified(podfile))
+	}
+}
+
+func TestGetWorkspaceProjectMap(t *testing.T) {
+	// ---------------------
+	// No workspace defined
+	t.Log("no workspace defined, no project in the pod folder -- ERROR")
+	{
+		tmpDir, err := pathutil.NormalizedOSTempDirPath("__utility_test_1__")
```
Comment (godrei) [inline utility/podfile_util_test.go]: cleanup tmp dir

**Pull request merged**